Task 1/Task1.2/Task1.2/AVG.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace ConsoleApp1
+{
+    class Program
+    {
+        
+        static void Main(string[] args)
+        {
+            string str = "Загрузка символов пропущена. Модуль оптимизирован, включен параметр отладчика Только мой код.";
+            char[] div = { ' ', ',', '.' };
+            string[] result = str.Split(div);
+            int len = result.Length;
+            int sum = 0;
+            for (int i = 0; i < result.Length; i++) {
+                if (result[i].Length == 0) len--;
+                sum += result[i].Length;
+            };
+            Console.WriteLine((double)sum/(double)len);//Not rounded.
```

Comment:
```suggestion
            Console.WriteLine(sum/len);//Not rounded.
```
Явное приведение здесь не требуется. Ну и в принципе явное приведение это не хорошо.

Reply:
Console.WriteLine((double)sum/(double)len);//7,2727272727272725
Console.WriteLine(sum/len);//7
Так что я неслучайно сделал приведение.